File: src/main/java/lach_01298/moreBees/recipes/RecipesCrafting.java
package lach_01298.moreBees.recipes;

import lach_01298.moreBees.Registry;
import lach_01298.moreBees.item.MoreBeesItems;
import net.minecraft.init.Items;

public class RecipesCrafting
{


	private static MoreBeesItems MBI;

	public static void registerRecipes(Registry registry)
	{
		Registry.addShapelessOreRecipe("diamond", Items.DIAMOND, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag, MoreBeesItems.DiamondFrag);
		Registry.addShapelessOreRecipe("emerald", Items.EMERALD, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag, MoreBeesItems.EmeraldFrag);
		//GameRegistry.addShapelessRecipe(new ItemStack(Items.NETHER_STAR),MBI.NetherFrag,MBI.NetherFrag,MBI.NetherFrag,MBI.NetherFrag);
	}



}

File: src/main/java/com/tencao/morebees/recipes/RecipesCrafting.kt
package com.tencao.morebees.recipes

import com.tencao.morebees.ObjRegistry
import com.tencao.morebees.events.MBRegistry
import net.minecraft.init.Items

object RecipesCrafting {

    fun registerRecipes() {
        MBRegistry.addShapelessOreRecipe("diamond", Items.DIAMOND, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag, ObjRegistry.DiamondFrag)
        MBRegistry.addShapelessOreRecipe("emerald", Items.EMERALD, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag, ObjRegistry.EmeraldFrag)
        //MBRegistry.addShapelessRecipe(new ItemStack(Items.NETHER_STAR),MBI.NetherFrag,MBI.NetherFrag,MBI.NetherFrag,MBI.NetherFrag);
    }
}